.docs/README.md

```diff
@@ -167,6 +167,50 @@ final class TestConsumer implements IConsumer
 }
 ```
 
+### Consuming messages in bulk
+
+Sometimes, you want to consume more messages at once, for this purpose, there is BulkConsumer.
+
+TestBulkConsumer.php
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Bunny\Message;
+use Contributte\RabbitMQ\Consumer\IConsumer;
+
+final class TestConsumer
+{
+
+    /**
+     * @param Message[] $messages
+     * @return array(delivery_tag => MESSAGE_STATUS)
+     */
+	public function consume(array $messages): array
+	{
+	    $return = [];
+	    $data = [];
+	    foreach($messages as $message) {
+	        $data[$message->deliveryTag] = json_decode($message->content);
+	    }
+	    
+        /**
+         * @todo bulk message action
+         */ 
+         
+         foreach(array_keys($data) as $tag) {
+            $return[$tag] = IConsumer::MESSAGE_ACK; // Or ::MESSAGE_NACK || ::MESSAGE_REJECT
+         }
+	    
+		return $return; 
+	}
+
+}
+```
+
+
```

Comment:
Could you please fix the indentation? Thanks

Reply:
done, spaces instead of tabs, sorry :)